src/sqlfluff/core/default_config.cfg

```diff
@@ -58,6 +58,47 @@ allow_scalar = True
 single_table_references = consistent
 unquoted_identifiers_policy = all
 
+# Rules can be grouped together for easy enabling/disabling
+[sqlfluff:rules:groups]
+core =
```

Comment:
Are any tests using this group of rules? I see the group_exclude and group_include tests. Seems like we should have a test for this group, since it's the one users are most likely to use.

Reply:
Right now, no but I could add a test for that